src/Driver.h

```diff
@@ -33,9 +27,9 @@ extern CProxy_TreeSpec treespec_subtrees;
 extern CProxy_TreeCanopy<CentroidData> centroid_calculator;
 extern CProxy_CacheManager<CentroidData> centroid_cache;
 extern CProxy_Resumer<CentroidData> centroid_resumer;
-extern CProxy_CountManager count_manager;
 
 namespace paratreet {
+  extern void preTraversalFn(CProxy_Driver<CentroidData>&, CProxy_CacheManager<CentroidData>& cache);
```

Comment:
Would there be any reason to add the iteration number here as well?

Reply:
I cant think of any but now that the function is clearly defined, the user can easily say "oh i can just manually edit this function to give me a new variable"